backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc

```diff
@@ -47,7 +47,6 @@ For more details, see xref:../../../backup_and_restore/application_backup_and_re
 
 include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
 
-include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
```

Comment:
I assume this is intentional, but mentioning just in case: this one is removed where the others are commented out